### PR TITLE
Fixed issues mentioned in git comments

### PR DESCRIPTION
**Pull request by charithag**
Fix for issues mentioned in https://github.com/wso2/carbon-device-mgt/pull/281

### CODE REVIEW

File: components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/modules/policy.js [hunk 1]
```diff
@@ -136,12 +136,12 @@ policyModule = function () {
         var mqttBrokerPort = mqttEndPointDeviceConfig.getPort();
         var mqttQueueEndpoint = mqttBrokerURL + ":" + mqttBrokerPort;
 
-        var mqttsenderClass = Packages.org.wso2.carbon.device.mgt.iot.mqtt.PolicyPush;
-        var mqttsender = new mqttsenderClass();
+        var mqttSenderClass = Packages.org.wso2.carbon.device.mgt.iot.mqtt.PolicyPush;
+        var mqttSender = new mqttSenderClass();
```
Comment (DilanUA) [inline components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/modules/policy.js:140]: It's not recommended to access any component level classes directly from jaggery runtime. 
What if the jaggery app is hosted as a separate webapp in a remote server with no direct java component class level access? Thus, it's always the best practice to make a REST API call to a backend server from jaggery end to perform the same.

File: components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/pages/cdmf.page.policies/policies.js [hunk 2]
```diff
@@ -35,15 +35,18 @@ function onRequest(context) {
...
         page["policyListToView"] = policyListToView;
         var policyCount = policyListToView.length;
-        if (policyCount == 0) {
+        switch (policyCount) {
+            case 0:
             page["policyListingStatusMsg"] = "No policy is available to be displayed.";
             page["saveNewPrioritiesButtonEnabled"] = false;
             page["noPolicy"] = true;
-        } else if (policyCount == 1) {
+                break;
```
Comment (DilanUA) [inline components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/pages/cdmf.page.policies/policies.js:43]: have some formating issues. indent the content in switch case properly.

File: components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/modules/policy.js (hunk 1 above)
Comment (charithag) [inline components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/modules/policy.js:140]: Ack, Fixed in commit https://github.com/wso2/carbon-device-mgt/pull/304/commits/d2daa0507eb9eb1b5d149fde50e305634e0f6b51

File: components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/pages/cdmf.page.policies/policies.js (hunk 2 above)
Comment (charithag) [inline components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/pages/cdmf.page.policies/policies.js:43]: Ack, Fixed in commit https://github.com/wso2/carbon-device-mgt/pull/304/commits/d2daa0507eb9eb1b5d149fde50e305634e0f6b51

**Pull request merged**